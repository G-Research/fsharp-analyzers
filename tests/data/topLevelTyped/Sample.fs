module Sample

let areEqual a b = a.Equals(b)

let a = areEqual 1 2

// let maxOf a b = if a > b then a else b
