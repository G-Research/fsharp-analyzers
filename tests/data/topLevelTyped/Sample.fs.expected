GRA-UNTYPEDAPI-001 | Warning | (3,4 - 3,12) | `areEqual` is not private, not fully typed and not used outside `A.fs`.Specify the return type. Add type annotation for parameter `a`
Add type annotation for parameter `b`.
GRA-UNTYPEDAPI-001 | Warning | (5,4 - 5,5) | `a` is not private, not fully typed and not used outside `A.fs`.Specify the return type.
